### PR TITLE
feat: apply SuperchainWETH contracts to forked networks

### PR DESCRIPTION
**Pull request by tremarkley**
Part of https://github.com/ethereum-optimism/supersim/issues/135

- Adds SuperchainWETH contracts to the interop applier package
- Enables loading env variables from .env
- Updates tests to use rpc urls from repo secrets 
  - This was needed because the public rpc urls were causing test failures

### CODE REVIEW

**Comment by tremarkley** [comment]
* **#148** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/supersim/148?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tremarkley and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: interop/applier.go [hunk 1]
```diff
@@ -28,6 +28,8 @@ var interopPredeploys = []common.Address{
 	predeploys.CrossL2InboxAddr,
 	predeploys.L2toL2CrossDomainMessengerAddr,
 	predeploys.L1BlockAddr,
+	common.HexToAddress("0x4200000000000000000000000000000000000024"),
+	common.HexToAddress("0x4200000000000000000000000000000000000025"),
```
Comment (tremarkley) [inline interop/applier.go:32]: will be able to import these from predeploys package once https://github.com/ethereum-optimism/optimism/pull/11927 merges

File: cmd/main.go [hunk 2]
```diff
@@ -59,7 +66,7 @@ func main() {
 		},
 	}
 
-	ctx := opio.WithInterruptBlocker(context.Background())
+	ctx := ctxinterrupt.WithSignalWaiterMain(context.Background())
```
Comment (tremarkley) [inline cmd/main.go:69]: needed to be updated based on refactor done in https://github.com/ethereum-optimism/optimism/pull/11511

File: interop/applier.go [hunk 3]
```diff
@@ -177,3 +187,15 @@ func waitMinedWithTicker(ctx context.Context, b bind.DeployBackend, tx *types.Tr
...
+	// Remove the "0x" prefix if it's present
+	if len(hexString) > 1 && hexString[:2] == "0x" {
+		hexString = hexString[2:]
+	}
+
+	bigInt := new(big.Int)
+	bigInt.SetString(hexString, 16)
+
+	return bigInt
+}
```
Comment (jakim929) [inline interop/applier.go:201]: nit: I think you might not need a helper function
 `new(big.Int).setBytes(common.FromHex(hexString))`

**Review by jakim929 — APPROVED** [review]
looks great! one nit

File: interop/applier.go (hunk 3 above)
Comment (tremarkley) [reply interop/applier.go]: oh nice, this is much better

**Pull request merged**